tasks/release-notes/release-notes.js

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
+/* eslint-disable no-undef */
+/* eslint-env node, es6*/
+
+const fs = require('fs')
+
+const { Octokit } = require('octokit')
+const yargs = require('yargs')
+
+// ------------------------
+
+const milestonesQuery = `
+  query($title: String!) {
+    repository(owner: "redwoodjs", name: "redwood") {
+      milestones(query: $title, first: 10) {
+        nodes {
+          title
+          id
+        }
+      }
+    }
+  }
+`
+
+const prsQuery = `
+  query($id: ID!) {
+    node(id: $id) {
+      ... on Milestone {
+        title
+        pullRequests(first: 100) {
+          nodes {
+            number
+            title
+            author {
+              login
+              url
+            }
+            url
+          }
+          totalCount
+          pageInfo {
+            hasNextPage
+          }
+        }
+      }
+    }
+  }
+`
+
+// a pr looks like...
+//
+// {
+//   "number": 2613,
+//   "title": "Scaffold Generator File Organization",
+//   "author": {
+//     "login": "cjreimer",
+//     "url": "https://github.com/cjreimer"
+//   },
+//   "url": "https://github.com/redwoodjs/redwood/pull/2613"
+// }
+const formatPR = (pr) =>
+  `- ${pr.title} [#${pr.number}](${pr.url}) by [@${pr.author.login}](${pr.author.url})`
```

Comment:
I personally have a preference for unformatted, e.g.:
`const formatPR = (pr) => `- ${pr.title} #${pr.number} by @${pr.author.login}`

But mine is a specific use-case:
- need to quickly read and group via copy/paste. markdown is challenging to read quickly (and copy/paste)
- I work in GitHub Release Notes draft, so the links are automatically added

Maybe I could read better I there was a space between lines. Open to options and preferences!

Reply:
> I work in GitHub Release Notes draft, so the links are automatically added

Since this is a thing, let's go with the formatting you suggested to make sure everything's readable